src/patternfly/components/Alert/examples/alert-warning-example.hbs

```diff
@@ -1,33 +1,37 @@
- {{#> alert pf-c-alert--Modifier="pf-m-warning" pf-c-alert--AriaLabel="Warning Notification"}}
-<div aria-hidden="true" class="pf-c-alert__icon">
-  <i class="fas fa-exclamation-triangle"></i>
-</div>
-<div class="pf-c-alert__body">
-  <h4 class="pf-c-alert__title">
-    <span class="pf-u-sr-only">Warning: </span> Warning notification title</h4>
-    <p>This is a description of the notification content.</p>
-</div>
+{{#> alert alert--modifier="pf-m-warning" alert--attribute='aria-label="Warning Notification"'}}
+  {{#> alert-icon alert-icon--warning="true"}}
+  {{/alert-icon}}
+  {{#> alert-body alert-HasDescription="true"}}
+    {{#> alert-title}}
+      {{#> sr}} Warning:{{/sr}}
+      Warning notification title
+    {{/alert-title}}
+  {{/alert-body}}
 {{/alert}}
 <br>
-<br> {{#> alert pf-c-alert--Modifier="pf-m-warning" pf-c-alert--AriaLabel="Warning Notification"}}
-<div aria-hidden="true" class="pf-c-alert__icon">
-  <i class="fas fa-exclamation-triangle"></i>
-</div>
-<div class="pf-c-alert__body">
-  <h4 class="pf-c-alert__title">
-    <span class="pf-u-sr-only">Warning: </span> Warning notification title</h4>
-</div>
-<div class="pf-c-alert__action">
-  {{#> button btnClass="pf-m-secondary"}} Button {{/button}}
-</div>
+{{#> alert alert--modifier="pf-m-warning" alert--attribute='aria-label="Warning Notification"'}}
+  {{#> alert-icon alert-icon--warning="true"}}
+  {{/alert-icon}}
+  {{#> alert-body}}
+    {{#> alert-title}}
+      {{#> sr}}Warning: {{/sr}}
+      Warning notification title
+    {{/alert-title}}
+  {{/alert-body}}
+  {{#> alert-action alert--attribute="pf-c-alert__action"}}
```

Comment:
looks like `alert--attribute` here is misplaced.

Reply:
resolved